ci/infra/testrunner/tests/test_deployments.py

```diff
@@ -0,0 +1,22 @@
+from platforms import Platform
+from skuba import Skuba
+from kubectl import Kubectl
+import pytest
+import time
+from timeout_decorator import timeout
+
+def test_nginx_deployment(setup, skuba, kubectl):
```

Comment:
are you also planning on having a jenkins pipeline as a consumer of this test?

Reply:
The idea is to have one nightly that runs all of this test scenarios